frontend/src/components/Pages/FrontPage/MissionOverview/ScheduleMissionDialog/SelectAndScheduleMissionsDialog.tsx

```diff
@@ -0,0 +1,128 @@
+import { Autocomplete, Button, Card, Dialog, Typography } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+import { useLanguageContext } from 'components/Contexts/LanguageContext'
+import { useState } from 'react'
+import { useInstallationContext } from 'components/Contexts/InstallationContext'
+import { Robot } from 'models/Robot'
+import { EchoMissionDefinition } from 'models/MissionDefinition'
+import { useRobotContext } from 'components/Contexts/RobotContext'
+import { BackendAPICaller } from 'api/ApiCaller'
+
+const StyledMissionDialog = styled.div`
+    display: flex;
+    justify-content: space-between;
+`
+const StyledAutoComplete = styled(Card)`
+    display: flex;
+    justify-content: center;
+    padding: 8px;
+    gap: 25px;
+    box-shadow: none;
+`
+const StyledMissionSection = styled.div`
+    display: flex;
+    margin-left: auto;
+    margin-right: 0;
+    gap: 10px;
+`
+const StyledDialog = styled(Dialog)`
+    display: flex;
+    padding: 1rem;
+    width: 320px;
+`
+
+interface ScheduleDialogProps {
+    echoMissions: Map<string, EchoMissionDefinition>
+    closeDialog: () => void
+    setLoadingMissionSet: (foo: (missionIds: Set<string>) => Set<string>) => void
+}
+
+export const SelectAndScheduleMissionsDialog = ({
```

Comment:
It took me a while with the name of this one to get the difference from the schedule mission dialog, I would suggest change it to SelectMissionsToScheduleDialog. I know it's a little thing but yeah